devolutions-gateway/src/http/http_server.rs

```diff
@@ -31,14 +37,40 @@ pub fn configure_http_server(config: Arc<Config>, jet_associations: JetAssociati
                     Some(auth_exclude_path),
                 )
                 .apply(LogMiddleware, vec!["/"], None)
+                .apply(
+                    SogarAuthMiddleware::new(config.clone()),
+                    vec![BLOB_PATH, BLOB_GET_LOCATION_PATH, UPLOAD_BLOB_PATH, MANIFEST_PATH],
+                    vec!["registry/oauth2/token"],
+                )
```

Comment:
Move that before the LogMiddleware (middlewares are executed in LIFO order)

Reply:
done